src/components/AnnotationPopover/AnnotationPopover.scss

```diff
@@ -75,17 +75,19 @@
     }
 
     .ba-popover-flipped {
-        path,
-        rect {
-            display: none;
-        }
-
         .ba-popover-caret {
             bottom: 0;
             transform: rotate(180deg);
         }
     }
 
+    .ba-point-annotation-marker.ba-popover-flipped {
```

Comment:
Can be nested in `.ba-popover-flipped` above via `&.ba-point-annotation-marker`.